.github/workflows/workflow.yml

```diff
@@ -15,20 +15,26 @@ jobs:
       fail-fast: false
       matrix:
         job:
+        - { os: macos-latest   , ocaml-version: 4.11.1 }
         - { os: macos-latest   , ocaml-version: 4.10.1 }
         - { os: macos-latest   , ocaml-version: 4.09.1 }
         - { os: macos-latest   , ocaml-version: 4.08.1 }
         - { os: macos-latest   , ocaml-version: 4.07.1 }
+        - { os: macos-latest   , ocaml-version: 4.05.0 }
+        - { os: ubuntu-latest  , ocaml-version: 4.11.1 }
         - { os: ubuntu-latest  , ocaml-version: 4.10.1 }
         - { os: ubuntu-latest  , ocaml-version: 4.10.0+musl+static+flambda }
         - { os: ubuntu-latest  , ocaml-version: 4.09.1 }
         - { os: ubuntu-latest  , ocaml-version: 4.08.1 }
         - { os: ubuntu-latest  , ocaml-version: 4.07.1 }
-        - { os: windows-latest , ocaml-version: 4.10.1+mingw64 }
```

Comment:
Why is this deleted?

Reply:
The Windows OCaml installation defaults to the MinGW 64-bit version, so, "4.10.1+mingw64" is the same as just "4.10.1".